3_extract.R

```diff
@@ -9,8 +9,14 @@ p3 <- list(
   # saving only the temperature predictions for each depth and ice flags
   tar_target(
     p3_gcm_glm_uncalibrated_output_feathers,
-    write_glm_output(p2_gcm_glm_uncalibrated_run_groups,
-                     outfile_template='3_extract/out/GLM_%s_%s.feather'),
+    {
+      output_files <- p2_gcm_glm_uncalibrated_run_groups %>% # Already grouped by site_id
+        group_by(driver) %>% # Also group by driver (GCM) for creating export files
+        group_map(~ write_glm_output(.x, outfile_template='3_extract/out/GLM_%s_%s.feather'),
```

Comment:
I keep forgetting about `group_map()`! I need to remember to use that one of these days ...